inc/classes/class-helper.php

```diff
@@ -83,7 +83,7 @@ public static function filter_input( $type, $variable_name, $filter = FILTER_DEF
 			 */
 			switch ( $filter ) {
 				case FILTER_SANITIZE_STRING:
-					$sanitized_variable = sanitize_text_field( filter_input( $type, $variable_name, $filter ) );
+					$sanitized_variable = filter_input( $type, $variable_name, $filter );
```

Comment:
Why removing `sanitize_text_field `?

Reply:
https://github.com/rtCamp/wp-google-login/pull/25#discussion_r372376643. Otherwise, for the state, we have to use FILTER_DEFAULT or pass a flag in this helper method to not run that variable through `sanitize_text_field`.

Reply:
I mean, You are removing `Helper::` from https://github.com/rtCamp/wp-google-login/pull/25/files/c8c28167657aa7fa69d81325fd840c126bf7f7a0#diff-b469e88c428b8984eaffd6ad3550cbebL313

And removing `sanitize_text_field ` too. Let me know the reason to make both these changes?

If we only remove `sanitize_text_field ` and dont remove `Helper::` from call part, then it should work i believe!

Reply:
I am just removing santizie_text_field from the code and not the Helper I have updated the code.